web/components/input/buttons/RadioGroup/RadioButton.sass

```diff
@@ -1,54 +1,49 @@
+.radio-group
+  @extend .flex, .wrap
```

Comment:
This is used here as well: https://github.com/nwhacks/nwhacks2019/blob/master/web/styles/decoration.sass#L5

That being said, this would be much better declared inline in the JSX

Reply:
Fixed